package.json

```diff
@@ -40,6 +40,11 @@
     "url": "https://github.com/openmrs/openmrs-esm-login/issues"
   },
   "dependencies": {
+    "@carbon/colors": "^10.19.0",
```

Comment:
I don't see this being used.

Reply:
yes cleaning this out , there was controversy  during selection.

Reply:
Yes sure this is going to be remevod.